README.md

```diff
@@ -49,6 +47,20 @@ To run this sample:
 6. Run the sample:
     - mvn clean compile exec:java
 
+## Use latest key vault SDK
+
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
```

Comment:
IIRC you have another PR that uses the same Key Vault content. My comments still apply. (I cannot find that PR at the moment.)